proof/crefine/RISCV64/IsolatedThreadAction.thy

```diff
@@ -1319,7 +1269,9 @@ lemma threadGet_isolatable:
 
 lemma setCurThread_isolatable:
   "thread_actions_isolatable idx (setCurThread t)"
-  by (simp add: setCurThread_def modify_isolatable)
+sorry (* Maybe no longer true, now that setCurThread contains the ready_qs_runnable assertion,
+which depends on the thread state, and could be modified by partial_overwrite *)
+(*  by (simp add: setCurThread_def modify_isolatable) *)
```

Comment:
Interesting. If you're fairly sure it is no longer true, we should turn this into an `oops` instead of `sorry`. (Also, for either, please tag with a `FIXME RT` comment, so we can count the overall to-do items in CRefine)

It could become fairly annoying for `seCurThread` to not be re-orderable. Do you think it'd be possible to remove the assertion?

Reply:
(It's perfectly fine to leave this as `oops` for now and continue sorrying the rest first. Maybe that'll also show how much impact its absence really has)

Reply:
Sure, I will `oops` this and see what happens. It's been a bit too long since I did anything to do with `ready_qs_runnable`, but I think its use is fairly limited and we might be able to improve things somehow